desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -107,19 +107,24 @@ def handle_on_link_shared(channel_id, message_ts, links):
 
     try:
       if path == '/hue/editor' and id_type == 'editor':
-        doc = Document2.objects.get(id=qid)
+        doc = Document2.objects.get(id=qid) if qid.isdigit() else Document2.objects.get(uuid=qid)
```

Comment:
Time to simplify uuid/if, same on line 121

query_id = {'uuid': .} if ... else {'id: ..}

then

get(**query_id)

and even 112 and more maybe